src/components/MenuItm.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import "./MenuItm.css";
+
+function MenuItm({ item }) {
```

Comment:
No need to skip one letter in name

And for the future might be helpful for components to have their own folders under the "components" folder, so the component code, styles, etc. is in one place

components/{component name folder}/{code, styles, etc}